frontend/package.json

```diff
@@ -36,9 +37,9 @@
       ".(ts|tsx|js|jsx)": "./node_modules/ts-jest/preprocessor.js"
     },
     "transformIgnorePatterns": [
-      "<rootDir>/node_modules/(?!lodash-es/.*)"
+      "<rootDir>/node_modules/(?!lodash-es|@console)"
     ],
-    "testRegex": "/__tests__/.*\\.(ts|tsx|js|jsx)$",
+    "testRegex": "(/__tests__/.*|\\.test)\\.(ts|tsx|js|jsx)$",
```

Comment:
Stick to using `spec` as that's the adopted file name pattern

Reply:
Agreed, will do.